content/docs/faq.md

```diff
@@ -210,3 +217,59 @@ Our deployment is running in [OpenShift
 Online](https://www.openshift.com/products/online/) which is using docker as a
 container engine, that's why we are using docker here and not
 [podman](https://github.com/containers/podman).
+
+If this is not enough, there is more! You can pull our production worker image
+and run packit commands in there to reproduce issues with parsing packit.yaml
+or interacting with the git repo.
+```
+$ docker pull docker.io/usercont/packit-worker:prod
+```
+
+There are multiple operations which happen in the worker:
+* cloning repositories
+* parsing packit.yaml
+* interacting with the git repo
+* interacting with other services
+
+This how we can create a SRPM in a very similar way packit service does it (ogr
+is one of the libraries powering packit):
+```
+$ docker run -it --rm -v $PWD:/ogr -w ogr docker.io/usercont/packit-worker:prod bash
```

Comment:
Suggestion: The "ogr" bit is not magic, right? So maybe more generally `-v $PWD:/src -w /src` to avoid confusion?